### DISCUSSION TITLE
评论无法显示，一直在转圈【hugo LoveIt】

### DISCUSSION THREAD

**Discussion by ZhengLiu-727** [walinejs/waline#735] [Q&A]
我使用的是hugo loveit主题，引入了waline评论，request 和 response都可以正常的发出和收到，其他都正常，就是在评论区无法显示具体的评论，一直在转圈，并且console有一个报错。访问[我的网站](https://www.liuzheng.site/first_post/)也可以看到这个现象

具体表现如图：
![](https://typora-us.oss-us-west-1.aliyuncs.com/20211229225156.png)

能够收到评论的数据：
![](https://typora-us.oss-us-west-1.aliyuncs.com/20211229225339.png)

console报错：Cannot read properties of null 
![](https://typora-us.oss-us-west-1.aliyuncs.com/20211229225524.png)

我个人认为是在客户端和hugo主题之间产生了冲突，但无法定位，希望大佬帮忙看一下

**Comment by BachWV** [comment]
同样的问题
![image](https://user-images.githubusercontent.com/21078868/179396866-0c54dc79-56d3-462c-a9a7-ce55d9d27532.png)

**Reply by BachWV** [reply]
已经解决了

**Reply by FDAW51** [reply]
你好能详细说一下这个问题怎么解决的吗？


---
Hello, can you explain in detail how to solve this problem?
